Smash_Combos.Core/Cqrs/Characters/CharacterResponse.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Smash_Combos.Domain.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Smash_Combos.Core.Cqrs.Characters
+{
+    public class CharacterResponse
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string VariableName { get; set; }
+        public int YPosition { get; set; }
+        public decimal ReleaseOrder { get; set; }
+        public List<Combo> Combos { get; set; }
```

Comment:
This actually made me realise I also made this mistake (copy paste whoops). We shouldn't use the domain entity as part of the response object, this will probably still cause the json loop error. You should create another class of "ComboDto" that contains just the information needed from the Combo entity, for the character response. i.e. if only 3 of the properties are used, then only get those 3.
You'll also have to create another map in the profile to make it work.

Reply:
If I understand correctly that means we'll have different ComboDtos for different requests, right?

Reply:
Yup that's exactly right